README.md

```diff
@@ -10,4 +10,9 @@ To install **kns** you can just run this command:
 ```bash
 curl https://raw.githubusercontent.com/blendle/kns/master/kns -o /usr/local/bin/kns && chmod +x $_
 ```
+Or use our homebrew tap:
+```bash
+$ brew tap blendle/blendle
+$ brew install kns
+```
```

Comment:
In READMEs I prefer to have the `tap` and `install` listed as separate commands so that it's clear that you need to tap into a repository, and then install the formula (new users won't understand that `brew install org/repo/formula` implies `brew tap && brew install`).